Tan_ShinYi/Assignments/Python_Fundamentals/Average_List.py

```diff
@@ -0,0 +1,12 @@
+a = [1, 2, 5, 10, 255, 3]
+total=0
+
+for i in range(len(a)):
+    total += a[i]
```

Comment:
good. in cases where you don't care about the indices, it's more efficient and simpler to iterate the elements directly, e.g.: `for ele in a: total+=ele`